notifier/github/comment.go

```diff
@@ -73,14 +72,57 @@ func (g *CommentService) DeleteDuplicates(title string) {
 
 func (g *CommentService) getDuplicates(title string) []*github.IssueComment {
 	var dup []*github.IssueComment
-	re := regexp.MustCompile(`(?m)^(\n+)?` + title + `\n+` + g.client.Config.PR.Message + `\n+`)
-
 	comments, _ := g.client.Comment.List(g.client.Config.PR.Number)
 	for _, comment := range comments {
-		if re.MatchString(*comment.Body) {
+		if g.bodyMatchString(title, *comment.Body) {
 			dup = append(dup, comment)
 		}
 	}
 
 	return dup
 }
+
+// regexp.MustCompile(`(m?)^(\n+)?` + title + `\n+` + g.client.Config.PR.Message + `\n+`)
```

Comment:
Awesome work! But I won't like to maintain this if I was the maintainer, because there's no concrete use-case which justifies this kind of engineering effort to speed up tfnotify.